.cspell.json

```
@@ -605,6 +605,7 @@
     "dxferp",
     "dyaw",
     "dylib",
+    "eagleye",
```

In this case, we prefer the formal style.

```suggestion
    "Eagleye",
```